frontend/src/views/edit-project/EditProjectView.tsx

```diff
@@ -109,6 +110,12 @@ export const EditProjectView = () => {
     if (displayedRepresentation && displayedRepresentation.id !== representationId) {
       const pathname = generatePath(routeMatch.path, { projectId, representationId: displayedRepresentation.id });
       history.push({ pathname });
+    } else if (!displayedRepresentation && performance.navigation.type !== PerformanceNavigation.TYPE_RELOAD) {
```

Comment:
We should not have to depend on the "performance" API here. I guess the issue (and fix) is similar to what you did for the OnboardArea bug, i.e. refine the state machine.